modules/bouncer_scriptworker/files/requirements.txt

```diff
@@ -3,7 +3,7 @@ PyYAML==3.12
 aiohttp==3.2.1
 arrow==0.12.1
 async_timeout==3.0.0
-bouncerscript==1.3.0  # puppet: nodownload
+bouncerscript==2.0.0  # puppet: nodownload
```

Comment:
Let's make sure we push this wheel to the internal pypi mirrors before merging this to `master`.